CONTRIBUTING.md

```diff
@@ -95,28 +95,64 @@ yarn dev --force
 
 > You can now hack in the cloned repository of `api-platform-admin`.
 
-#### Running Admin through Storybook
+#### Running Admin Through Storybook
 
-If you don't have an existing API Platform application, or don't want to use `yarn link`, you can use one of the bundled example APIs, and visualize the admin through [Storybook](https://storybook.js.org/).
+If you do not have an existing project, you can contribute by visualizing directly the changes you made the sources through [Storybook](https://storybook.js.org/). 
+
+This development stack consists of two Docker containers: 
+- `pwa`: containing the `<Admin>` sources and Storybook;
+- `php`: holding the API sources.
+
+Additionally, this method allows testing the integration between Api-platform and the `admin` component by writing stories, scenarios and tests.
 
 Install everything:
 
 ```shell
 docker compose up
 ```
 
-Go to http://localhost, accept the self-signed certificate, and see the running Admin.
+Before accessing the Storybook instance, make sure to go to https://localhost to accept the self-signed certificate. Once it's done, you'll see the API documentation running on a customized version of Swagger UI.
```

Comment:
Actually, because of this line https://github.com/marmelab/api-platform-admin/blob/chore/add-jwt-backend-and-stories/api/frankenphp/docker-entrypoint.sh#L5, the `vendor` folder was not updated. Removing it to force updating the dependencies fixed the issue.